docs/reference/controls/itemscontrol.md

```diff
@@ -17,9 +18,34 @@ To see the full list of _Avalonia UI_ built-in repeating data controls, see [her
 
 You will probably use these properties most often:
 
-<table><thead><tr><th width="316">Property</th><th>Description</th></tr></thead><tbody><tr><td><code>ItemsSource</code></td><td>The bound collection that is used as the data source for the control.</td></tr><tr><td><code>ItemsControl.ItemTemplate</code></td><td>Attached property element to contain the data template for an individual item. </td></tr></tbody></table>
-
-## Example
+<table>
+  <thead>
+    <tr>
+      <th width="316">Property</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>ItemsSource</code></td>
+      <td>The bound collection that is used as the data source for the control.</td>
+    </tr>
+    <tr>
+      <td><code>ItemsControl.ItemTemplate</code></td>
+      <td>Attached property element to contain the data template for an individual item.</td>
```

Comment:
I know you did you not write this, but the item template property is not an attached property. I am not sure why it says this in the first place. Attached properties are a very specific thing.

Reply:
no problem, what would be the correct description? Should I just remove the 2 leading words "Attached property"

Reply:
I would reword it entirely to something like "The item template, contains a DataTemplate which will be applied to individual items and can be used to change how items look". That way it better explains why someone would even use it and what it does.